### ISSUE TITLE
Can we use it for Visual Studio Code

### ISSUE THREAD

**Issue by eliassal** [madskristensen/WebCompiler#474]
Hi, is it possible to use it with Visual Studio Code? If yes, how?
Thanks in advance

**Comment by mackcoding** [comment]
No, and the developer hasn't updated this project in years.

**Comment by rangav** [comment]
I am also interested in this extension for vs code

**Comment by marylaureColibri** [comment]
Yes you can run the webcompiler (in VS Pro) from your VS Code ^^

You can use a "run on save" extension (VS Code) that run a command line to run Webcompiler.exe

like :
`"command": "c:\\users\\userOnThisComputer\\appdata\\local\\microsoft\\visualstudio\\visualStudioVersionNumberOnYourComputer\\Extensions\\webCompilerVersionNumberOnYourCompute\\WebCompiler.exe \"${workspaceFolder}/yourProjectOnyourComputer/compilerconfig.json\" *.LESS",`
